sites/github.styl

```diff
@@ -269,14 +272,37 @@ a.filter-item
 .branch-action-state-dirty
 .branch-action-state-unstable
 .branch-action-state-unknown
-    .branch-action-icon
+.branch-action-icon
```

Comment:
Why did you remove the indentation here?  IIUC it will now make all `.branch-action-icon` icons yellow.

Reply:
Ah, this is a mistake, I didn't know what the difference was, correcting!

Reply:
No prob.  Indentation with selectors is equivalent to putting them on the same line separated by a space in regular CSS, so this selector only applies to `.branch-action-icon` classed items that are children of the `-state-*` selectors above it.